src/fuzzers/packet-fuzzer.cc

```diff
@@ -18,6 +18,9 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size)
 	if ( ! fb.Valid() )
 		return 0;
 
+	if ( fb.ChunkCount() > 64 )
```

Comment:
I see that Jon already merged this, but would it be better to move this check into `FuzzBuffer::Valid()` to avoid having to repeat it in every fuzzer?

Reply:
Never mind, he apparently already did it 😄